testing/e2e/app/routes/Table/spec.ts

```diff
@@ -0,0 +1,191 @@
+
+      // other columns should not change
+      expect(newWidths[0]).toBe(initialWidths[0]);
+      expect(newWidths[2]).toBe(initialWidths[2]);
+      expect(newWidths[3]).toBe(initialWidths[3]);
+    }
+  });
+
+  test('should respect min width', async ({ page }) => {
+    const minWidth0 = 50;
+    const minWidth1 = 300;
+
+    await page.goto(`/Table?minWidth=${minWidth0}&minWidth=${minWidth1}`);
+
+    // resize first column
+    {
+      await resizeColumn({ index: 0, delta: -100, page, step: true });
+      const newWidths = await getColumnWidths(page);
+      expect(newWidths[0]).toBe(minWidth0);
+    }
+
+    // resize second column
+    {
+      await resizeColumn({ index: 1, delta: -600, page, step: true });
+      const newWidths = await getColumnWidths(page);
+      expect(newWidths[1]).toBe(minWidth1);
+    }
+
+    // resize third column (has implicit min width)
+    {
+      await resizeColumn({ index: 2, delta: -800, page, step: true });
+      const newWidths = await getColumnWidths(page);
+      expect(newWidths[2]).toBe(72); // 72 is the implicit min width
+    }
+  });
+
+  test('should respect max width', async ({ page }) => {
+    const maxWidth0 = 150;
+    const maxWidth1 = 300;
+
+    await page.goto(`/Table?maxWidth=${maxWidth0}&maxWidth=${maxWidth1}`);
+
+    // resize first column
+    {
+      await resizeColumn({ index: 0, delta: +100, page, step: true });
+      const newWidths = await getColumnWidths(page);
+      expect(newWidths[0]).toBe(maxWidth0);
+    }
+
+    // resize second column
+    {
+      await resizeColumn({ index: 1, delta: +100, page, step: true });
+      const newWidths = await getColumnWidths(page);
+      expect(newWidths[1]).toBe(maxWidth1);
+    }
+  });
+
+  test('should respect disableResizing', async ({ page }) => {
+    await page.goto('/Table?disableResizing=true');
+
+    // resize first column
+    {
+      const initialWidths = await getColumnWidths(page);
+
+      const delta = +50;
+      await resizeColumn({ index: 0, delta, page });
+
+      const newWidths = await getColumnWidths(page);
+      expect(newWidths[0]).toBe(initialWidths[0] + delta);
+      expect(newWidths[1]).toBe(initialWidths[1]); // should not change
+    }
+  });
+
+  // #region Helpers for column resizing tests
+  const resizeColumn = async (options: {
+    index: number;
+    delta: number;
+    page: Page;
+    step?: boolean;
+  }) => {
+    const { index, delta, page, step = false } = options;
+
+    const resizer = page.getByRole('separator').nth(index);
+    const resizerBox = (await resizer.boundingBox())!;
+
+    await resizer.hover({
+      position: { x: resizerBox.width / 2, y: resizerBox.height / 2 },
+    });
+    await page.mouse.down();
+    await page.mouse.move(
+      Math.max(1, resizerBox.x + delta + resizerBox.width / 2),
+      resizerBox.y + resizerBox.height / 2,
+      { steps: step ? Math.abs(delta) : 5 },
+    );
+    await page.mouse.up();
```

Comment:
Should we also test for touch events, similar to the old "shoulf handle resize with touch" test?

Reply:
I'll have to see if it's possible to do a meaningful test for touch events in playwright. TBD

Reply:
Yeah I couldn't write anything meaningful. I tried manually constructing touch events (which already kinda defeats the purpose of testing), but even that didn't work.

<details><summary>Code</summary>

```tsx
test('should work with touch events', async ({ page }) => {
  await page.goto('/Table');

  const initialWidths = await getColumnWidths(page);

  await page.evaluate(() => {
    const resizer = document.querySelector('[role="separator"]')!;
    const resizerBox = resizer.getBoundingClientRect();

    resizer.dispatchEvent(
      new TouchEvent('touchstart', {
        touches: [
          new Touch({
            identifier: Date.now(),
            target: resizer,
            clientX: resizerBox.x,
            clientY: resizerBox.y,
          } as any),
        ],
      }),
    );
  });

  await page.evaluate(() => {
    const resizer = document.querySelector('[role="separator"]')!;
    const resizerBox = resizer.getBoundingClientRect();
    resizer.dispatchEvent(
      new TouchEvent('touchmove', {
        touches: [
          new Touch({
            identifier: Date.now(),
            target: resizer,
            clientX: resizerBox.x + 100,
            clientY: resizerBox.y ,
          } as any),
        ],
      }),
    );
  });

  await page.evaluate(() => {
    const resizer = document.querySelector('[role="separator"]')!;
    resizer.dispatchEvent(new TouchEvent('touchend'));
  });

  const newWidths = await getColumnWidths(page);
  expect(newWidths[0]).toBe(initialWidths[0] + 100);
});
```

</p>
</details>

fwiw, I think it's fine to leave it untested. The logic for mouse vs touch isn't all that different, plus I verified that it works on my phone, and I haven't touched (no pun intended) the code in this PR.